.github/workflows/jest.yml

```
@@ -0,0 +1,27 @@
+jobs:
```

actionlint

"on" section is missing in workflow

(syntax-check)